requirements/common.txt

```diff
@@ -0,0 +1,6 @@
+requests==2.24.0  # https://requests.readthedocs.io/en/master/
+connexion[swagger-ui]==2.7.0  # https://github.com/zalando/connexion
+Flask==1.1.2  # https://flask.palletsprojects.com/en/1.1.x/e
+Flask-Wtf==0.14.3  # https://flask-wtf.readthedocs.io/en/stable/
+
+selenium==3.141.0  # https://selenium-python.readthedocs.io/
```

Comment:
I believe selenium is for testing alone, so probably shall be in the -dev requirememnts